.github/BOTMETA.yml

```diff
@@ -703,6 +703,8 @@ files:
     maintainers: bgaifullin cloudnull
   $modules/remote_management/lxca/:
     maintainers: navalkp prabhosa
+  $modules/remote_management/lenovoxcc/:
```

Comment:
Can you move this before the previous entry, so it's sorted alphabetically? Thanks :)

Reply:
Sorry, I haven't notice it. Code is updated.